src/engine/TPA.cc

```diff
@@ -1403,74 +1425,77 @@ VerificationResult TransitionSystemNetworkManager::solve() && {
     initNetwork();
 
     auto current = getChainStart();
```

Comment:
Note that now there does not have to be any `chain start`, because there is no chain anymore. In fact, now the exploration should start by traversing all the edges from `Entry` point of the graph.